### ISSUE TITLE
Running into the same #311 issue again: this time with the package `fuzzy`

### ISSUE THREAD

**Issue by farzadbekran** [agentm/project-m36#319]
Same situation: missing a symbol, but this time it's from the dependency of `fuzzy`, namely `monoid-subclasses`

**Comment by farzadbekran** [comment]
No still the same problem:
```
project-m36-server: src/DB/ConstraintFunctions.o: unknown symbol `monoidzmsubclasseszm1zi0zi1zmGVl1nDJj2pLCZZya9zzwWBfh_DataziMonoidziTextual_zdfTextualMonoidText0_closure'
./start-db.sh: line 3: 213966 Segmentation fault      (core dumped) stack exec -- project-m36-server -n shopper-db -d shopper-db --fsync
```

**Comment by farzadbekran** [comment]
Solved in #327
